testbed/tests/scenarios.go

```diff
@@ -32,6 +36,7 @@ import (
 
 var (
 	performanceResultsSummary testbed.TestResultsSummary = &testbed.PerformanceResults{}
+	itemsPerIntervalEnvVar                               = "ITEMS_PER_INTERVAL"
```

Comment:
1. This should be a `const` not `var`.
2. Why is this needed? Why is it more important to make this parameter available as en var, but not some other parameters, like for example data points per second?